tag/feature.py

```diff
@@ -27,7 +28,7 @@ class Feature(object):
     'gene'
     >>> feature.start, feature.end
     (999, 7500)
-    >>> feature.score is None
+    >>> feature.score.value is None
```

Comment:
Maybe `feature.score` should be `feature._score`, and then define a property function to return the score value. That makes for a nicer interface.